lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -13,70 +13,83 @@
+                request=request,
+            ),
+            'course_updates': reverse(
                 'course-updates-list',
                 kwargs={'course_id': course_id},
-                request=request
-            )
-            course_handouts_url = reverse(
+                request=request,
+            ),
+            'course_handouts': reverse(
                 'course-handouts-list',
                 kwargs={'course_id': course_id},
-                request=request
-            )
-            discussion_url = reverse(
+                request=request,
+            ),
+            'discussion_url': reverse(
                 'discussion_course',
                 kwargs={'course_id': course_id},
-                request=request
-            ) if course_overview.is_discussion_tab_enabled() else None
-        else:
-            video_outline_url = None
-            course_updates_url = None
-            course_handouts_url = None
-            discussion_url = None
+                request=request,
+            ) if course_overview.is_discussion_tab_enabled() else None,
 
-        if course_overview.advertised_start is not None:
-            start_type = "string"
-            start_display = course_overview.advertised_start
-        elif course_overview.start != DEFAULT_START_DATE:
-            start_type = "timestamp"
-            start_display = defaultfilters.date(course_overview.start, "DATE_FORMAT")
-        else:
-            start_type = "empty"
-            start_display = None
+            'video_outline': reverse(
+                'video-summary-list',
+                kwargs={'course_id': course_id},
+                request=request,
+            ),
 
-        return {
-            "id": course_id,
-            "name": course_overview.display_name,
-            "number": course_overview.display_number_with_default,
-            "org": course_overview.display_org_with_default,
-            "start": course_overview.start,
-            "start_display": start_display,
-            "start_type": start_type,
-            "end": course_overview.end,
-            "course_image": course_overview.course_image_url,
-            "social_urls": {
-                "facebook": course_overview.facebook_url,
+            # Note: The following 2 should be deprecated.
+            'social_urls': {
+                'facebook': course_overview.facebook_url,
             },
-            "latest_updates": {
-                "video": None
+            'latest_updates': {
+                'video': None
             },
-            "video_outline": video_outline_url,
-            "course_updates": course_updates_url,
-            "course_handouts": course_handouts_url,
-            "discussion_url": discussion_url,
-            "subscription_id": course_overview.clean_id(padding_char='_'),
-            "courseware_access": has_access(request.user, 'load_mobile', course_overview).to_json() if request else None
         }
```

Comment:
Are we sure there won't be any scenario when request param is not available; just wondering why was it put there initially.